openpose_ros/KeyPoint[] body_parts

